halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/HaEchoValidator.java

```diff
@@ -38,7 +38,7 @@ public void validate(ConfigProblemSetBuilder p, DeploymentEnvironment n) {
       p.addProblem(
           Problem.Severity.WARNING,
           "High Availability (HA) is enabled for echo, but found custom sizing for the main service (this setting will be ignored). "
-              + "With HA enabled, the service is split into multiple sub-services (echo-scheduler, echo-scheduler). You need to update the component sizing for each sub-service, individually.");
+              + "With HA enabled, the service is split into multiple sub-services (echo-scheduler, echo-worker). You need to update the component sizing for each sub-service, individually.");
```

Comment:
[note]
You can see the sub-service names:
https://github.com/spinnaker/halyard/blob/d8af98d3dddf3dd420f64534895a1dd7ee85efae/halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/HaEchoValidator.java#L45-L46

Reply:
How about specifying the full service name in the warning message as it is configured in customSizing: `spin-echo-scheduler` and `spin-echo-worker`?

Reply:
Oh, sure! 
I'll do it 👍

Reply:
I've done 4fdeae2